playwright/_impl/_waiter.py

```diff
@@ -16,15 +16,15 @@
 import math
 import uuid
 from asyncio.tasks import Task
-from typing import Any, Callable, List, Tuple
+from typing import Any, Callable, List, Tuple, Union
 
 from pyee import EventEmitter
 
-from playwright._impl._api_types import Error, TimeoutError
 from playwright._impl._connection import ChannelOwner
+from playwright._impl._errors import Error, TimeoutError
 
 
-class WaitHelper:
+class Waiter:
```

Comment:
I like the old name better, it is less ambiguous as it has less to do with restaurants business.

Reply:
sure, but keep the ports aligned! :/